### DISCUSSION TITLE
Since last update of all components error message  Section 'autotune_tmc stepper_a' is not a valid config section

### DISCUSSION THREAD

**Discussion by Ejay-3D** [Guilouz/Klipper-Flsun-Speeder-Pad#145] [Q&A]
Since last update Section 'autotune_tmc stepper_a' is not a valid config section
I use your printer.cfg file 

Can you tell more or solve this issue  ? 
for now I commented those lines 

[klippy (11).log](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/files/13627844/klippy.11.log)

**Comment by lin-ycv** [comment]
press check for update on update manager and update/recover the TMC module, if it says up to date, ssh and run the command `wget -O - https://raw.githubusercontent.com/andrewmcgr/klipper_tmc_autotune/main/install.sh | bash`
